charts/harvester-csi-driver/templates/deployment.yaml

```diff
@@ -64,6 +64,7 @@ spec:
             - --timeout=1m50s
             - --leader-election
             - --leader-election-namespace=$(POD_NAMESPACE)
+            - --default-fstype=ext4
```

Comment:
The argument is appended to the attacher pod. It should be used by the csi-provisioner